packages/quantic/force-app/main/default/lwc/quanticTooltip/quanticTooltip.css

```diff
@@ -1,24 +1,57 @@
+:host {
+  --x-translation: var(--adapted-x-translation, 0px);
+  --max-width: var(--adapted-max-width, 20rem);
+}
+
 .tooltip__content {
   left: 50%;
-  transform: translate(-50%, -100%);
+  transition: transform 0.05s linear 0.05s, opacity 0.1s linear 0s, visibility 0.1s linear 0s;
```

Comment:
I think this is causing the <> under the tooltip to be visible for a fraction of a second when the tooltip is first rendered.
Can you apply the same transition to the <> as well?

Reply:
Yea I can adjust the transition of the small arrow